index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+  <header>
+    <nav class="wapi">
+      <ul class="nav-menu">
+          <li class="nav-item">
+              <a href="index.html" class="nav-link">Home</a>
+              <a href="About.html" class="nav-link">About</a>
+          </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </nav>
+  <div class="site-content">
+    <div class="wanted">      <ul>
+        <li class="jof"><img class="kla" src="Assets/bg/f2.png" alt="f2"></li>
+      </ul>
+      <img class="kla" src="Assets/bg/f2.png" alt="f2">
+      <img class="kla" src="Assets/bg/twiter.png" alt="twitter">
+        <ul class="wina">
+        <li>English</li>
+        <li>Mypage</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <nav class="navbar">
+      <div>
+      <img class="lolo" src="Assets/CC_Global_Summit_2015 logo.png" alt="CC_Global_Summit_2015" />
+    </div>
+    <div>
+      <ul class="menu">
+        <li><a id="a" href="About.html">About</a></li>
+        <li class="item"><a href="#program">Program</a></li>
+        <li class="item"><a href="#jopn">Join</a></li>
+        <li class="item"><a href="#sponser">Sponser</a></li>
+        <li class="item"><a href="#news">News</a></li></li>
+        <li class="itoa">CC Campaign</li>
+      </ul>
+    </div>
+    </nav>
+</header>
+  <body>
+           <div class="contents">
+                    <h2 class="contents-head">"Hello! Sharing World"</h2>
+                    <h1 class="contain">CREATIVE COMMONS<BR>GLOBAL SUMMIT<BR>2015</h1>
+                      <div class="load">
+                        <P class="paragraph">
+                            A joyful celebration in the value of openness and  sharing, ceating <br> a positive change with people from over 80 countries is taking place, in october Korea
+                        </P>
+                        <h3 class="date">2015.10.15(THU) ~ 16(FRI)</h3>
+                        <p >@ National Museam of Korea, Art Center Nabi and more</p>
+                      </div>
+                </div>
+            </div>
+            <section class="card-section">
+              <div class="card-container">
+                  <h1 class="box">Main Programs</h1>
```

Comment:
**[FIX BEFORE MERGE]**

- [this issue is not yet resolved.](https://github.com/maserekafelex/Final-project/blob/6680cbfef57157c41a9ca8c865be1badf075e0cc/index.html#L67). KIndly resolve before you merge. I see this is your last submission, since you've tried making the rest changes, I'd overlook this, but please, fix this before you merge.